modules/account-lib/src/coin/avaxc/keyPair.ts

```diff
@@ -20,9 +20,9 @@ export class KeyPair extends Secp256k1ExtendedKeyPair {
     super(source);
```

Comment:
Shouldn't this `if` go into the base class now that it looks the same for almost all coins?

Reply:
Looks like xrp doesn't have the `if` statement and `eth` has one that's a bit different. Still maybe this logic could be moved into a member function called in the constructor which the two coins above override.

Reply:
there is room for a lot more cleanup, but this PR was more about getting rid of utxolib